src/main/java/net/mcreator/element/types/interfaces/IOtherModElementsDependent.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.types.interfaces;
+
+import net.mcreator.element.parts.procedure.Procedure;
+import net.mcreator.generator.mapping.MappableElement;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public interface IOtherModElementsDependent {

```

Comment:
Those classes may not be entirely clear to the code-users, so some minimal JavaDOC should be added

Reply:
Would that be both minimal and enough?